### PR TITLE
Bug 1795707: improve notification drawer performance

### PR DESCRIPTION
**Pull request by spadgett**
* Don't lazy-load component since it will always load. This prevents the black box over the entire page during load
* Fix problem where `useEffect` hook constantly fires
* Don't return a new object each call to `notificationStateToProps` when alerts are empty
* Fix incorrect FirehoseResult type

/assign @jcaianirh @benjaminapetersen

### CODE REVIEW

**Comment by spadgett** [comment]
/hold

I'm still seeing an issue, will debug

File: analyze.sh [hunk 1]
```diff
@@ -14,11 +14,12 @@ if [ -d "$ARTIFACT_DIR" ]; then
   cp public/dist/report.html "${ARTIFACT_DIR}"
 fi
 
+MAX_BYTES=2621440 # ~2.5 MiB
```
Comment (spadgett) [inline analyze.sh]: You can't do floating point calculations in bash (2.5 * 1024 * 1024), so I just hard-coded the value

**Comment by spadgett** [comment]
Rebased, @jcaianirh @benjaminapetersen another look?

File: analyze.sh (hunk 1 above)
Comment (jcaianirh) [reply analyze.sh]: @spadgett i ran into this too...was wondering how you wanted this to be modified...

**Pull request merged**